network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/typeform_block.html

```diff
@@ -0,0 +1,5 @@
+<script src="https://embed.typeform.com/embed.js" ></script>
```

Comment:
Turns out this is not how we should be loading css/js for streamfields: https://stackoverflow.com/a/43561880/740553

Reply:
@Pomax the link you included seems to be the way to inject JS to Wagtail **admin** but not the published page. I'm gonna keep this `<script>` line.

Reply:
ahhhhh damn that is a very good point. I swear there was a way to inject a script "only once" this way too.... O_o